aws/rust-runtime/aws-types/Cargo.toml

```diff
@@ -9,6 +9,12 @@ license = "Apache-2.0"
 
 [dependencies]
 lazy_static = "1"
+tracing = "0.1.26"
```

Comment:
Should be less specific on the version number?

Reply:
yeah good call, when we use tracing as a library `0.1` is probably what we want